resources/js/app.js

```diff
@@ -1,2 +1,12 @@
-import 'Vendor/rapidez/core/resources/js/app.js'
-import 'Vendor/rapidez/account/resources/js/callbacks.js'
\ No newline at end of file
+import "Vendor/rapidez/core/resources/js/app.js";
+import "Vendor/rapidez/account/resources/js/callbacks.js";
+import "Vendor/rapidez/wishlist/resources/js/wishlist.js";
```

Comment:
What about these, can we also find something for this in case they're not found?